fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -56,13 +56,16 @@ private const val IMAGE_HEIGHT_RATIO_SMALL = 0.28f
  * A composable for displaying onboarding page content.
  *
  * @param pageState [OnboardingPageState] The page content that's displayed.
+ * @param pagePosition The current page position.
  * @param modifier The modifier to be applied to the Composable.
  * @param onDismiss Invoked when the user clicks the close button. This defaults to null. When null,
  * it doesn't show the close button.
  */
 @Composable
+@Suppress("LongMethod")
 fun OnboardingPage(
     pageState: OnboardingPageState,
+    pagePosition: Int? = null,
```

Comment:
This is a common component, this shouldn't care about the page position. See [this](https://github.com/mozilla-mobile/firefox-android/pull/5178/files#r1453242485) for alternate approach